### PR TITLE
Add instruction to run as service

### PR DESCRIPTION
**Pull request by shirshak55**
1. Useful for a long-running task

Thanks.

### CODE REVIEW

**Comment by Enter-tainer** [comment]
There are some systemd unit files in example/systemd, maybe you can simply reuse them.

see https://github.com/rapiz1/rathole/blob/main/examples/systemd/rathole@.service

**Comment by rapiz1** [comment]
Hey thanks for the contribution. As @Enter-tainer pointed out, there are already examples for systemd units. I think what can be improved here is to add a `README.md` under `examples/systemd` about how to use them, and add a link in the main `README.md` to that.

**Comment by shirshak55** [comment]
@rapiz1 thanks for the feedback. Would you mind reviewing again :)

File: README.md [hunk 1]
```diff
@@ -64,6 +64,28 @@ Then run:
...
 So you can `ssh myserver.com:5202` to ssh to your NAS.
 
+4. If you want to run server as service you can follow the steps given below:
+
+##### On Linux (Systemd):
+
+a. Create a service file:
+```bash
+wget https://github.com/rapiz1/rathole/blob/main/examples/systemd/rathole@.service # download the file
+sudo cp rathole@.service /lib/systemd/system/rathole.service
```
Comment (rapiz1) [inline README.md:74]: `@` mark has special meaning for systemd and shouldn't be removed at will. You may want to checkout https://superuser.com/questions/393423/the-symbol-and-systemctl-and-vsftpd and man pages.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,28 @@ Then run:
 
 So you can `ssh myserver.com:5202` to ssh to your NAS.
 
+4. If you want to run server as service you can follow the steps given below:
+
```
Comment (rapiz1) [inline README.md:68]: I would like this section in `examples/systemd/README.md` and put a link to it here. Running as a service is a common use case but requires efforts and understanding to do it right. That means it's not easy to explain it clearly in quick start, which should be short and concise. Also some people don't need at all.

**Review by rapiz1 — CHANGES_REQUESTED** [review]

**Comment by shirshak55** [comment]
@rapiz1 Looks good now?

**Comment by rapiz1** [comment]
@shirshak55 I see nothing changed since my last review. Did you forget to push?

**Comment by shirshak55** [comment]
> @shirshak55 I see nothing changed since my last review. Did you forget to push?

yea should be good now.

**Review by rapiz1 — APPROVED** [review]
LGTM

**Pull request merged**